### PR TITLE
Added Educational website

### LINKED ISSUE #3481: Landing page for an educational website
**Describe the project you want to add with tech stack**
Landing page for an educational website

**Screenshots (optional)**

image

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Landing page for an educational website #3481

Close #3481 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`

image

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
@pranjay-poddar @MohitGupta121 Please merge this

**Comment by PoulavBhowmick03** [comment]
@pranjay-poddar @MohitGupta121 @TusharKesarwani

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: @PoulavBhowmick03 please don't change this.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 fixed

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**